tests/test_modeling_tf_transfo_xl.py

```diff
@@ -131,6 +141,12 @@ def create_and_check_transfo_xl_lm_head(self, config, input_ids_1, input_ids_2,
             [(self.mem_len, self.batch_size, self.hidden_size)] * self.num_hidden_layers,
         )
 
+    def create_and_check_transfo_xl_for_sequence_classification(self, config, input_ids_1, input_ids_2, lm_labels):
+        config.num_labels = self.num_labels
```

Comment:
@LysandreJik I have defined 'num_labels' here.
Now,I'll move it to 'prepare_config_and_inputs'